lib/Controller/ChatController.php

```diff
@@ -160,6 +160,9 @@ protected function getActorInfo(string $actorDisplayName = ''): array {
 			if ($actorDisplayName) {
 				$this->guestManager->updateName($this->room, $this->participant, $actorDisplayName);
 			}
+		} elseif ($this->userId === MatterbridgeManager::BRIDGE_BOT_USERID && $actorDisplayName) {
+			$actorType = Attendee::ACTOR_BRIDGED;
```

Comment:
Missing from documentation:
https://github.com/nextcloud/spreed/blob/master/docs/constants.md#actor-types-of-chat-messages